src/middlewared/middlewared/plugins/system.py

```diff
@@ -993,6 +1025,8 @@ def run(self):
 
     await middleware.call('system.general.set_language')
 
+    await middleware.call('system.update_timeout_value')
```

Comment:
Was this tested? I dont think you can call initshutdown plugin during system setup()?
Perhaps use the system boot event?

Reply:
I think I am confusing things, this should be fine.